cli/azd/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 0.5.1-beta.1 (2023-01-12)
```

Comment:
In the past, we've been bumping the `-beta.X` segment of the pre-release version. Are we bumping `.X-beta.0` to highlight the bigger change instead?

Reply:
Went back to follow the previous pattern as you suggested.  This is now 0.5.0-beta.2